packages/dev/core/src/Misc/webRequest.fetch.ts

```diff
@@ -0,0 +1,37 @@
+import { WebRequest } from "./webRequest";
+
+/**
+ * Fetches a resource from the network
+ * @param url defines the url to fetch the resource from
+ * @param options defines the options to use when fetching the resource
+ * @returns a promise that resolves when the resource is fetched
+ * @internal
+ */
+export function _FetchAsync(

```

Comment:
Seems like this is kind of like a fetch polyfill. Couldn't we actually just use `fetch` directly and use an existing `fetch` polyfill that is built on top of `XmlHttpRequest` in cases where we need it (e.g. Babylon Native)? @bghgary

Reply:
Agreed. And it is internal so we can remove it when native will have fetch support